docs/spec_draft.md

```diff
@@ -343,6 +344,66 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_training
+
+### Semantics
+
+Computes mean and variance across batch and spatial dimensions and normalizes
+the `operand` tensor, for each feature in the `feature_index` dimension and
+produces `output`, `batch_mean` and `batch_var` tensors. Refer
+[Batch Normalization Paper](https://arxiv.org/abs/1502.03167) for the detailed
+algorithm.
```

Comment:
![image](https://user-images.githubusercontent.com/29868128/199116107-72ad440d-a25a-4f7f-945b-ab7f691d2d68.png)

How about saying this Batch Norm Transform from the paper

Reply:
@sdasgup3 reworded the semantics

Reply:
As discussed offline, the current plan is to reformulate the Semantics section similarly to `batch_norm_inference`: https://github.com/openxla/stablehlo/pull/377#discussion_r1013499398. I'll wait for the reformulation before reviewing the section.